.github/workflows/run_monitoring.yml

```diff
@@ -16,7 +14,7 @@ on:
 # https://github.com/google-github-actions/auth#setup
 
 jobs:
-  inference:
```

Comment:
was this causing the problems?

Reply:
Nope, just the wrong name in the UI (Which can be confusing) :)